<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmAdministrarBodegas</class>
 <widget class="QDialog" name="frmAdministrarBodegas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrar Bodegas</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Numero de bodega a eliminar:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEliminarBod">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>EliminarBodega</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCrear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Bodega</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbBodegas">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
